### ISSUE TITLE
AttributeError returned when Selection Custom Field in POST body

### ISSUE THREAD

**Issue by WilliamMarti** [netbox-community/netbox#5029]
### Environment
* Python version: 3.6.9
* NetBox version: 2.9.2

### Steps to Reproduce
1.  POST request to the `/api/ipam/prefixes/{id}/available-ips/` endpoint
2. POST request has a **Selection** custom field for example:

```
"custom_fields": {
    "ipam_DNSZone": 20,
}
```

Expected an IP address to be assigned.  I have my production Netbox instance running v2.8.8 where this same request works.  Note that I had a test instance running v2.9-beta1 where I believe this was working as well.  This request is also working when I leave out my custom field listed above altogether.  

Instead I am getting the error:

```
<class 'AttributeError'>

Got AttributeError when attempting to get a value for field `value` on serializer `CustomFieldChoiceSerializer`.
The serializer field might be named incorrectly and not match any attribute or key on the `int` instance.
Original exception text was: 'int' object has no attribute 'pk'.
```

If there is anything else I can provide let me know

**Comment by WilliamMarti** [comment]
I am running into this error when hitting the REST API, so I am just using Insomnia to make the above POST.  

Here is a python example of hitting the API that would generate the error (few things sanitized).  Only requirement would be having a custom field for the IP Address object type that is a Selection.    

```
import requests

url = "https://netbox.example.com/api/ipam/prefixes/6794/available-ips/"

payload = "{\n\t\"description\": \"testserver1\",\n\t\"custom_fields\": {\n\t\t\"selection_custom_field\": 20\n\t}\n}"
headers = {
    'authorization': "Token token",
    'content-type': "application/json",
    'accept': "application/json"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)
```

**Comment by jeremystretch** [comment]
This report needs to include

1. The complete list of steps someone else can take to reproduce the problem, beginning with the creation of the custom field; and
2. The raw REST API request being sent (e.g. using `curl`)

**Comment by jeremystretch** [comment]
I'm not able to reproduce this issue on v2.9.2. Do you by chance have a webhook defined for the IPAddress model? The error looks similar to #5081.

**Comment by WilliamMarti** [comment]
I do indeed have a webhook for the IPAddress model.  Just ran another test of the `/ipam/prefixes/11913/available-ips/` endpoint after disabling the webhook, and it worked.  Re-enabling the IPAddress webhook I am seeing the same error as above.

**Comment by jeremystretch** [comment]
@WilliamMarti Thanks. Definitely sounds like #5081 then. Will be fixed in v2.9.3.
